Slack.Automation/Promact.Core.Repository/TaskMailRepository/ITaskMailRepository.cs

```diff
@@ -25,46 +25,47 @@ public interface ITaskMailRepository
         /// <param name="answer"></param>
         /// <param name="userId"></param>
         /// <returns>questionText in string format containing question statement</returns>
-        Task<string> QuestionAndAnswer(string userName, string answer,string userId);
+        Task<string> QuestionAndAnswerAsync(string answer,string userId);
         
         /// <summary>
         /// This method use to fetch the task mail details.
         /// </summary>
-        /// <param name="UserId"></param>
-        /// <param name="UserRole"></param>
-        /// <param name="UserName"></param>
-        /// <param name="LoginId"></param>
-        /// <returns>task mail</returns>
-        Task<List<TaskMailUserAc>> TaskMailDetailsReport(string UserId,string UserRole,string UserName,string LoginId);
+        /// <param name="id"></param>
+        /// <param name="role"></param>
+        /// <param name="name"></param>
+        /// <param name="loginId"></param>
+        /// <returns></returns>
+        Task<List<TaskMailReportAc>> TaskMailDetailsReportAsync(string id,string role,string name,string loginId);
         /// <summary>
         /// This method getting list of Employees
         /// </summary>
-        /// <param name="UserId"></param>
-        /// <returns>list of employees</returns>
-        Task<List<TaskMailUserAc>> GetAllEmployee(string UserId);
+        /// <param name="id"></param>
+        /// <returns></returns>
+        Task<List<TaskMailReportAc>> GetAllEmployeeAsync(string id);
```

Comment:
It is not clear from name. what is the purpose of GetAllEmployees. It looks like it returns all employees. Please improve naming.